keps/sig-cloud-provider/991-cloud-controller-migration/README.md

```diff
@@ -163,29 +169,50 @@ type ControllerLeaderConfiguration struct {
 	// E.g. service-controller, route-controller, cloud-node-controller, etc
 	Name string `json:"name"`
 
-	// Component is the name of the component in which the controller should be running.
+	// Component is the name of the component in which the controller will be running.
 	// E.g. kube-controller-manager, cloud-controller-manager, etc
 	Component string `json:"component"`
 }
 ```
+#### Default LeaderMigrationConfiguration
+
+The `controller-manager` will provide `kube-controller-manager` and `cloud-controller-manager`
```

Comment:
What does the generic `controller-manager` mean in this context?  A specific repository?

Reply:
Nice catch, it's `staging/controller-manager`, updating.